### PR TITLE
Test on Node.js v0.10.x, v0.12.x, v4.x, and v5.x

### PR DESCRIPTION
**Pull request by ntwb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "iojs"
```
Comment (ljharb) [inline .travis.yml]: there's no need to remove iojs, support for it isn't dropped.

**Comment by ntwb** [comment]
Added back `iojs`

**Comment by Raynos** [comment]
lgtm.

**Comment by ntwb** [comment]
:+1:

**Pull request merged**